### DISCUSSION TITLE
Airplay and Slimproto issue

### DISCUSSION THREAD

**Discussion by webvision411** [music-assistant/support#1510] [Q&A]
hello, I'm running MA in Home Assistant and have amplipi up2stream amps. It works fine with UPNP but with both airplay and slimproto it discovers the players but says "the player is currently (not) available". Any ideas how to troubleshoot? nothing in the logs except 1 warning:
WARNING (MainThread) [music_assistant.webserver] [140093081458128] Invalid command: players/all

**Comment by OzGav** [comment]
First guess is that it is a networking issue. Have you been through the troubleshooting faq?
